app/src/main/AndroidManifest.xml

```diff
@@ -53,6 +53,8 @@
             android:name=".task.StarActivity"
             android:screenOrientation="fullSensor" />
         <activity android:name=".task.LessonCompleteActivity"></activity>
+        <activity android:name=".authentication.CameraViewActivity">
```

Comment:
Authentication stands in here for the complete set of functions which are needed for the face recognition. Starting with the process of taking images, storing them if faces are recognized, extracting features, start the training with the features and use the trained data to authenticate faces.